### ISSUE TITLE
SUGGESTION: Folder Specific Project Name

### ISSUE THREAD

**Issue by Lycrios** [guiguan/file-header#41]
I just love this plugin. It would be nice to have like a `.file-header` file that allows users to specify variables for specific projects. I am constantly switching between projects for different clients so it'd be nice to just open the folder and have that projects name auto-loaded into the `settings`.

**Comment by tommygaessler** [comment]
I agree with @Lycrios. Also being able to disable auto update/add fille header for certain projects based on their paths would be nice as well.

**Comment by liron-tiledmedia** [comment]
Would be great I agree... I like this addons a lot and after a few days I got here to ask the same thing :D

**Comment by Shinrai** [comment]
Not exactly what you are looking for. However my pull request will auto detect the project name if you open the project in Atom and then will use that name to fill in the file-header template.
https://github.com/guiguan/file-header/pull/65

**Comment by Igor1306** [comment]
Great feature, It would be great.
